egs/librispeech/ASR/conformer_ctc/transformer.py

```diff
@@ -301,6 +302,7 @@ def decoder_forward(
 
         return decoder_loss
 
+    @torch.jit.export
     def decoder_nll(
```

Comment:
> Mm, I imagine this somehow inspects the forward() function of the model? Would we still be able to use functions like decoder_nll()

@danpovey 
As long as you use the decorator `@torch.jit.export`, you can access it.